index.html

```diff
@@ -10,7 +10,8 @@
 </head>
 
 <body>
-  <h2>Tic Tac Toe</h2>
+  <h2>Tic Tac Toe: Epic Version</h2>
```

Comment:
If you want it to say something like "EPIC Tic-Tac-Toe!!!!!!!!" in the browser tab you can stick that in the `<title>` tag on line 8.